packages/utilities/src/ProcessUtils.ts

```diff
@@ -126,22 +126,23 @@ export class ProcessUtils {
      * the `{envVariablePrefix}_EDITOR` environment variable if specified.
      * @param filePath File path to edit
      */
-    public static async openInEditor(filePath: string) {
-        let editor;
-        if (ImperativeConfig.instance.loadedConfig.envVariablePrefix != null) {
+    public static openInEditor(filePath: string, editorOpt?: string, sync?: boolean) {
```

Comment:
I like that you enabled your enhancements in a non-breaking way. I think that we should add @param documentation for the new editorOpt and sync parameters.

Reply:
In addition to the previously requested changes, I think you will have to merge master into this branch and confirm that the tests pass before you can merge this PR.